ansible/roles/common/tasks/setup-wifi.yml

```diff
@@ -49,5 +61,5 @@
     daemon_reload: yes
     masked: no
   with_items:
-    - udhcpd
+    - dhcpcd
```

Comment:
Should this be dhcpcd5? Like in the vars.yaml is recorded as a variable?

Reply:
no is correct.
dhcpcd5 is the package name
dhcpcd is the service name